src/Avalonia.Controls/ContextMenu.cs

```diff
@@ -10,11 +10,38 @@
 using Avalonia.Interactivity;
 using Avalonia.Layout;
 using Avalonia.Styling;
+using Avalonia.VisualTree;
 
 #nullable enable
 
 namespace Avalonia.Controls
 {
+    /// <summary>
+    /// Provides data for the
+    /// <see cref="E:Avalonia.Controls.ContextMenu.Opened" />
+    /// event.
+    /// </summary>
+    public class ContextMenuEventArgs : RoutedEventArgs
+    {
+        PointerEventArgs Pointer;
+
+        /// <summary>
+        /// Initializes a new instance of the ContextMenuEventArgs
+        /// class.
+        /// </summary>
+        internal ContextMenuEventArgs(PointerEventArgs pointer)
+        {
+            Pointer = pointer;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the ContextMenuEventArgs
+        /// class.
+        /// </summary>
+        public Point? GetPosition(IVisual? relativeTo)
```

Comment:
I believe it should be mentioned in the docs summary, that GetPosition returns null, when context menu was opened without pointer interaction?

Reply:
UWP uses TryGet pattern in ContextRequestedEventArgs that I am going to port as well. Does it make sense to apply it here?
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.input.contextrequestedeventargs.trygetposition?view=winrt-20348#Windows_UI_Xaml_Input_ContextRequestedEventArgs_TryGetPosition_Windows_UI_Xaml_UIElement_Windows_Foundation_Point__